services/auth/token/lambdas/authorize.js

```diff
@@ -1,6 +1,7 @@
 import to from 'await-to-js';
-import { verifyToken } from '../../../../libs/token';
+
 import generateIAMPolicy from '../helpers/generateIAMPolicy';
+import { verifyToken } from '../../../../libs/token';
 
 export async function main(event) {
```

Comment:
Missing comment JS Doc Comment, can fix this. not crucial for this PR, since it's earlier approved code.